sqrl-io/sqrl-io-file/src/main/java/com/datasqrl/io/impl/file/FileDataSystemConfig.java

```diff
@@ -27,6 +30,10 @@ public class FileDataSystemConfig implements Serializable {
   @Default
   String filenamePattern = DEFAULT_FILENAME_PATTERN;
 
+  @Default
+  @JsonProperty("source.monitor-interval")
+  String source$$monitor$interval = "10s";
```

Comment:
Why did you make this a nested property?
We should probably come up with a naming convention for properties so we don't confuse users with different styles (e.g. camelCase, dashes, underscores)

Reply:
This is flink's table api config option for files. Some configuration options should be passthrough options, especially for the table api. This is particular example may be too much of a stretch since it's not passing through, but generally speaking we should support the full range of native configuration options.